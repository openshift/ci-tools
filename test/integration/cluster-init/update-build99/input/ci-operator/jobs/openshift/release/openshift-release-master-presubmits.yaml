presubmits:
  openshift/release:
    - agent: kubernetes
      always_run: true
      branches:
        - master
      cluster: app.ci
      context: ci/build-farm/build99-dry
      decorate: true
      labels:
        ci.openshift.io/build-farm: build99
        ci.openshift.io/generator: cluster-init
        pj-rehearse.openshift.io/can-be-rehearsed: "true"
      name: pull-ci-openshift-release-master-build99-dry
      rerun_command: /test build99-dry
      spec:
        containers:
          - args:
              - --config-dir
              - clusters/build-clusters/build99
              - --kubeconfig
              - /etc/build-farm-credentials/kubeconfig
              - --as=
              - --extra-arg=true
            command:
              - applyconfig
            env:
              - name: HOME
                value: /tmp
              - name: build99_id
                valueFrom:
                  secretKeyRef:
                    key: build99-id
                    name: build99-dex-oidc
              - name: github_client_id
                valueFrom:
                  secretKeyRef:
                    key: build01_github_client_id
                    name: build-farm-credentials
              - name: slack_api_url
                valueFrom:
                  secretKeyRef:
                    key: url
                    name: ci-slack-api-url
            image: registry.ci.openshift.org/ci/applyconfig:latest
            imagePullPolicy: Always
            name: ""
            resources:
              requests:
                cpu: 10m
            volumeMounts:
              - mountPath: /etc/build-farm-credentials
                name: build-farm-credentials
                readOnly: true
              - mountPath: /tmp
                name: tmp
        serviceAccountName: config-updater
        volumes:
          - name: build-farm-credentials
            secret:
              items:
                - key: sa.config-updater.build99.config
                  path: kubeconfig
              secretName: config-updater
          - emptyDir: {}
            name: tmp
      trigger: (?m)^/test( | .* )build99-dry,?($|\s.*)