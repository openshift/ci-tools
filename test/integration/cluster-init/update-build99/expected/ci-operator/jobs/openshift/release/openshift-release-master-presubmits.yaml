presubmits:
  openshift/release:
  - agent: kubernetes
    always_run: false
    branches:
    - ^master$
    - ^master-
    cluster: app.ci
    context: ci/build-farm/build99-dry
    decorate: true
    labels:
      ci.openshift.io/build-farm: build99
      ci.openshift.io/generator: cluster-init
      pj-rehearse.openshift.io/can-be-rehearsed: "true"
    name: pull-ci-openshift-release-master-build99-dry
    rerun_command: /test build99-dry
    run_if_changed: ^clusters/.*
    spec:
      containers:
      - args:
        - --config-dir=clusters/build-clusters/build99
        - --as=
        - --kubeconfig=/etc/build-farm-credentials/kubeconfig
        command:
        - applyconfig
        env:
        - name: HOME
          value: /tmp
        - name: build99_id
          valueFrom:
            secretKeyRef:
              key: build99-id
              name: build99-dex-oidc
        image: registry.ci.openshift.org/ci/applyconfig:latest
        imagePullPolicy: Always
        name: ""
        resources:
          requests:
            cpu: 10m
        volumeMounts:
        - mountPath: /etc/build-farm-credentials
          name: build-farm-credentials
          readOnly: true
        - mountPath: /tmp
          name: tmp
      serviceAccountName: config-updater
      volumes:
      - name: build-farm-credentials
        secret:
          items:
          - key: sa.config-updater.build99.config
            path: kubeconfig
          secretName: config-updater
      - emptyDir: {}
        name: tmp
    trigger: (?m)^/test( | .* )build99-dry,?($|\s.*)
